src/index.js

```diff
@@ -0,0 +1,101 @@
+const skyLeft = {
+  Sunny: '☀️🌤🌞🌤☀️',
+  Cloudy: '🌫☁️🌥⛅️🌤',
+  Rainy: '🌈⛈🌧🌧🌦',
+  Snowy: '🌨❄️🌨❄️🌨',
+};
+
+const skyRight = {
+  Sunny: '☀️🌤🌞🌤☀️',
+  Cloudy: '🌤⛅️🌥☁️🌫',
+  Rainy: '🌦🌧🌧⛈💧',
+  Snowy: '🌨❄️🌨❄️🌨',
+};
+
+const skyColor = {
+  Sunny: 'skyblue',
+  Cloudy: 'lightgrey',
+  Rainy: 'lightblue',
+  Snowy: 'lightsteelblue',
+};
+
+const skySelector = document.getElementById('sky__selector');
```

Comment:
Remember that it's a best practice to avoid attempting to retrieve element references until after we're certain that the DOM has completed loading. We do this by registering our code as an event handler for the `DOMContentLoaded` on the `document` instance.